src/main/resources/config/application.yml

```diff
@@ -69,7 +69,7 @@ artemis:
                     # possible overrides: maven, gradle
                     default: "ls1tum/artemis-maven-template:java17-21"
                 empty:
-                    default: "ls1tum/artemis-maven-template:java17-21"
```

Comment:
Why does the PR change the default docker image here?

Reply:
I didn't want to imply that the tests have to be Java based, so I went with something more basic. You would want to use a custom image anyway.

Reply:
I agree. Nevertheless, I would avoid not needed config changes as it might change workflows for instructors and also it increases the amount of used images.

Reply:
This is a good point. But a workflow using the default image of `Empty` and requiring Maven without using the `Java` template would be highly unusual.
Most of the other images are based on `ubuntu` down the line, so I wouldn't expect much of an increase in disk usage.